tests/test_module.py

```diff
@@ -159,9 +159,9 @@ def test_sanity(self):
 
             expected_result = {
               'totalSamples': 1500L,
-              'memoryUsage': 29022L,
+              'memoryUsage': 4238L,
```

Comment:
you will need to support both chunk types, take a look at my branch to how I changed the testing to accommodate both.